pkg/util/cloudfoundry/bbscache.go

```diff
@@ -211,11 +211,11 @@ func (bc *BBSCache) readData() {
 	}()
 	wg.Wait()
 	if errActual != nil {
-		log.Errorf("Failed reading Actual LRP data from BBS API: %s", errActual.Error())
+		_ = log.Errorf("Failed reading Actual LRP data from BBS API: %s", errActual.Error())
 		return
 	}
 	if errDesired != nil {
-		log.Errorf("Failed reading Desired LRP data from BBS API: %s", errDesired.Error())
+		_ = log.Errorf("Failed reading Desired LRP data from BBS API: %s", errDesired.Error())
```

Comment:
Why do we do this ?

Reply:
it is just to remove some linting warnings

Reply:
Those should be ignore by the [linter](https://github.com/DataDog/datadog-agent/blob/master/.golangci.yml#L62). We don't catch+ignore the error in the rest of the code.

What linter was complaining ? What is locally or in the CI ?

Reply:
No it was my IDE locally. I figured since I was editing this file, there was no harm in making this file green in my IDE.
Is this a problem ? I can remove if so

Reply:
It's just that's it's useless and make the code less readable. And we don't to is anywhere else in the code.
In this case I think less/simpler is better.
Especially since the configuration do skip them.

Reply:
OK removed.
My take on this, `log.Errorf` shouldn't return an error at all though, it's a bit weird to have a call to log return an error.
I realise it makes for a small shortcut when we want to log an error and return it too, but it doesn't feel like it's the main use case. Just my two cents 🙂 .

Reply:
I agree but it comes from the underlying lib `seelog` https://github.com/cihub/seelog/blob/master/log.go#L231